Packs/FWToCDLMonitoring/IncidentFields/incidentfield-panorama_integration.json

```diff
@@ -0,0 +1,30 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "panoramaintegration",
+    "closeForm": false,
+    "content": true,
+    "description": "name of the Panorama integration",
```

Comment:
panorama isstance name? or id of the integration? the default name of it is `Panorama`.

Reply:
The Panorama integration could be used to execute commands not only on Panorama devices but also on PANOS firewalls. As so, it's very common to have several of these integrations configured but only one of them will be connected to a Panorama management console while the others will be connected to classical firewalls. That's why I'm asking for the name of the Panorama instance connected to a Panorama device.